enterprise/app/controllers/super_admin/response_sources_controller.rb

```diff
@@ -41,4 +41,52 @@ class SuperAdmin::ResponseSourcesController < SuperAdmin::ApplicationController
 
   # See https://administrate-demo.herokuapp.com/customizing_controller_actions
   # for more information
+
+  include ActionView::Helpers::TagHelper
+  include ActionView::Context
+
+  helper_method :render_vue_component
+
+  def render_vue_component(component_name, props = {})
+    html_options = {
+      id: 'app',
+      data: {
+        component_name: component_name,
+        props: props.to_json
+      }
+    }
+    content_tag(:div, '', html_options)
+  end
```

Comment:
@sojan-official Can you please check if you can move this to base controller? If so, the render method can be used anywhere in the code.